src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.targets

```diff
@@ -264,8 +264,8 @@
       Condition="'$(SkipVersionGeneration)' != 'true' AND '$(VersionPropsImported)' != 'true'">
 
     <!-- ############################### -->
-    <Exec Command="git rev-parse --abbrev-ref HEAD 2>&amp;1" StandardOutputImportance="Low" IgnoreExitCode="true" IgnoreStandardErrorWarningFormat="true" ConsoleToMSBuild="true" Condition="'$(GitBranchName)' == ''">
-      <Output TaskParameter="ConsoleOutput" PropertyName="GitBranchName" />
+    <Exec Command="git describe --all HEAD 2>&amp;1" StandardOutputImportance="Low" IgnoreExitCode="true" IgnoreStandardErrorWarningFormat="true" ConsoleToMSBuild="true" Condition="'$(RawGitBranchName)' == ''">
```

Comment:
We'll have to use this new variable name in the source-build workaround: [repos/dir.targets#L7-L8](https://github.com/dotnet/source-build/blob/b8d175fa98bb4b52f61a212ee99bfec9327d828b/repos/dir.targets#L7-L8). (BuildTools-side fix tracked by https://github.com/dotnet/buildtools/issues/1980.)

Reply:
Thanks for the pointer. Should we just rename the property back to the old name in BuildTools?

Reply:
Nevermind I think I will just need to update the workaround in source-build.

Reply:
Sounds good, I think if you're going to make BuildTools-side changes, might as well fix https://github.com/dotnet/buildtools/issues/1980 fully. (Shouldn't be a very complicated change.)